src/Validator/Product.php

```diff
@@ -41,9 +59,47 @@ public function validateRequest(RequestInterface $request): bool
         $urlKey = $this->getPathFrontName($request);
         $productCollection = $this->productCollection->clear();
         $productCollection->addAttributeToFilter('url_key', $urlKey);
+
         $ids = $productCollection->getAllIds();
         $productId = reset($ids);
 
-        return !!$productId;
+        if (!$productId) return false;
```

Comment:
Please organize the code accordingly to PSR-2 https://www.php-fig.org/psr/psr-2/